bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/entity/generic/GenericClassification.kt

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package org.eclipse.tractusx.bpdm.gate.entity.generic
+
+import jakarta.persistence.*
+import org.eclipse.tractusx.bpdm.common.model.BaseEntity
+
+// only is called generic since other entity it this name already exists and will be removed on the future
+@Entity
+@Table(name = "generic_classifications")
+class GenericClassification(
+    @Column(name = "value")
+    var value: String?,
+
+    @Column(name = "code")
+    var code: String?,
+
+    @Column(name = "type")
+    var type: String?,
```

Comment:
I think it should be 
`@Enumerated(EnumType.STRING)
    val type: ClassificationType,` 
like in LegalEntityClassification